guides/wide-ep-lws/inferencepool-pd-override.values.yaml

```diff
@@ -0,0 +1,59 @@
+inferenceExtension:
```

Comment:
I don't expect we need to maintain another values.yaml here, we should be able to reuse the pd-values.yaml entirely, or perhaps with a couple overrides via `helm apply --set`. If we are blocked by the DP change, I think we should just wait.